packages/modules-sdk/src/remote-query.ts

```diff
@@ -86,34 +86,40 @@ export class RemoteQuery {
     relations: string[]
     args: Record<string, unknown[]>
   } {
+    const dataCopy = JSON.parse(JSON.stringify(data))
```

Comment:
Maybe we can name this better? queryConfig or something like that?

Reply:
Improved a bit further than that 👍